docs/.pages

```diff
@@ -1,7 +1,7 @@
 nav:
   - Introduction: index.md
   - installation
-  - tutorials
+  - simulation-tutorials
```

Comment:
@LalithVipulananthan I think changing the page title is enough. So how about keeping here as-is?
Ref: https://docs.ros.org/en/rolling/Tutorials.html

```suggestion
  - tutorials
```

Reply:
Okay. Presumably I would also need change the folder name back to `tutorials`? If so, then the PR will just be the page title change, so can I add the other changes that I had in mind for `tutorials/index.md` to this PR?

Reply:
> Presumably I would also need change the folder name back to tutorials?

Yes, please. :pray: 

> so can I add the other changes that I had in mind for tutorials/index.md to this PR?

Sure!